### PR TITLE
Add Pituitary gland example function

### PR DESCRIPTION
**Pull request by WolfByttner**

### CODE REVIEW

**Review by MaddyThakker — APPROVED** [review]
LGTM!

File: traja/dataset/pituitary_gland.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    inoise = 0 # noise*w #TODO fix
+
+    dv = -(ica+ik+inoise+ileak)/Cm
+    dn = (phik-n)/taun
+    df = (phif-f)/taubk
+    dc = -ff*(alpha*ica+kc*c)
+    return (dv, dn, df, dc)
+
+
+def compute_pituitary_gland_df_from_parameters(downsample_rate,
```
Comment (justinshenk) [inline traja/dataset/pituitary_gland.py:73]: This method name is too long, and the purpose is not clear. Please add a docstring for the method and shorten the name.

File: traja/dataset/pituitary_gland.py [hunk 2]
```diff
@@ -64,14 +64,36 @@ def pituitary_ode(w, t, p):
 
 def compute_pituitary_gland_df_from_parameters(downsample_rate,
```
Comment (justinshenk) [inline traja/dataset/pituitary_gland.py:73]: Not clear how this is different than `pituitary_ode`. Can we give it a shorter name?

File: traja/dataset/pituitary_gland.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    df['gbk'] = gbk
+    df['gl'] = gl
+    df['kc'] = kc
+    df = df.iloc[::downsample_rate, :]
+    # df = df.drop(columns=['t', 'ikdrx', 'ibkx'])
+
+    return df
+
+
+def create_latin_hypercube_sampled_pituitary_df(downsample_rate=100, samples=1000):
```
Comment (justinshenk) [inline traja/dataset/pituitary_gland.py:133]: maybe remove `df` from method name and also `create`, like `lhs_pituitary() or pituitary_lhs()`. Also add method docstring with return types eg, `-> DataFrame, int`.

File: traja/dataset/pituitary_gland.py [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+from numpy import exp
+from numba import jit
+from scipy.integrate import odeint
+from pyDOE2 import lhs
+
+
+# PyTest will not compute coverage correctly for @jit-compiled code.
+# Thus we must explicitly suppress the coverage check.
+@jit
+def pituitary_ode(w, t, p):  # pragma: no cover
```
Comment (justinshenk) [inline traja/dataset/pituitary_gland.py:12]: maybe add type hints for return values `-> Tuple[float, float, float, float]` with `from typing import Tuple` at the top

**Comment by justinshenk** [comment]
@WolfByttner I see it's already merged. Please let me know what you think about the suggestions.

**Pull request merged**